docs/datastore/spanner.md

```diff
@@ -108,7 +108,7 @@ gcloud projects add-iam-policy-binding $PROJECT_ID --member="serviceAccount:[SA_
       host: 0.0.0.0
       datastore:
         # Example for Spanner
-        kind: "spanner-gsql"
+        kind: "spanner"
```

Comment:
```suggestion
        kind: "spanner"
        # kind: "spanner-gsql"
```

Is there a preference towards spanner-pg vs spanner-gsql? Else, we might be able to do it this way.